pkgs/applications/audio/musikcube/default.nix

```diff
@@ -76,6 +76,11 @@ stdenv.mkDerivation rec {
     "-DDISABLE_STRIP=true"
   ];
 
+  postFixup = if stdenv.hostPlatform.isDarwin then ''
+    install_name_tool -add_rpath $out/share/${pname} $out/share//${pname}/${pname}
```

Comment:
Did you mean to have double slashes?

Reply:
I did not, thanks for pointing this out, @anderse.
I also seized the opportunity to improve this Darwin specific `postFixup` a bit.